server/src/Dao/RevisionDao.php

```diff
@@ -17,23 +19,18 @@ class RevisionDao extends AbstractDao {
 	public function getUserRevisionsInCommonPages(
 		$users, $startDate = null, $endDate = null, $limit = 50, $continue = null
 	) {
-		// build subquery to find common pages between users
-		$subQuery = $this->conn->createQueryBuilder();
-		$subQuery->select( 'rev_page' )
-			->from( 'revision_userindex' )
-			->where( 'rev_user_text in (:users)' )
-			->groupBy( 'rev_page' )
-			->having( 'count(distinct rev_user) > 1' );
+		$pages = $this->getUsersCommonPages( $users );
```

Comment:
I thought doing this would cause PHP to run out of memory?

Reply:
it did before but I was fetching a lot more info from the tables, so far, it seems to work for the 'malfunctioning' examples we were testing with. Maybe I should try to figure out what are the limits here.

Reply:
yeah that would be a good idea, at least just to know.

Reply:
We're you able to find out the limit(s)?

Reply:
Yes, 
> The default memory limit for grid engine webservice jobs is 4G. For Kubernetes the default limit is 2G for most runtimes (Java's limit is 4G). 

and php limit is set to -1. We should be fine